ddtrace/contrib/asgi/middleware.py

```diff
@@ -81,6 +81,9 @@ def __init__(
         self.handle_exception_span = handle_exception_span
         self.span_modifier = span_modifier
 
+        # Disable distributed tracing for possible downstream applications.
+        config.django.distributed_tracing_enabled = False
```

Comment:
This feels really icky but due to the config API not being able to tell us if a value is user-defined or not we can't really get a cleaner solution.

Ideally I'd like to do the following the Django integration:

```python
if config.asgi.distributed_tracing and config.django.distributed_tracing is None:
    config.django.distributed_tracing_enabled = False
```